src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/entity/Review.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity
+
+import io.hypersistence.utils.hibernate.type.array.EnumArrayType
+import io.hypersistence.utils.hibernate.type.array.ListArrayType
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
+import org.hibernate.annotations.Parameter
+import org.hibernate.annotations.SoftDelete
+import org.hibernate.annotations.Type
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.config.CsipRequestContext
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.entity.event.GenericCsipEvent
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.AffectedComponent
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.AuditEventAction
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.DomainEventType.ATTENDEE_CREATED
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.enumeration.ReviewAction
+import uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request.CreateAttendeeRequest
+import java.time.LocalDate
+import java.util.UUID
+
+@Entity
+@Table(name = "review")
+@SoftDelete
+@EntityListeners(AuditedEntityListener::class, UpdateParentEntityListener::class)
+class Review(
+  @ManyToOne(fetch = FetchType.LAZY, optional = false)
+  @JoinColumn(name = "plan_id")
+  val plan: Plan,
+
+  val reviewSequence: Int,
+  val reviewDate: LocalDate?,
+  val recordedBy: String,
+  val recordedByDisplayName: String,
+  val nextReviewDate: LocalDate?,
+  val csipClosedDate: LocalDate?,
+  val summary: String?,
+
+  @Type(ListArrayType::class, parameters = [Parameter(name = EnumArrayType.SQL_ARRAY_TYPE, value = "varchar")])
+  val actions: Set<ReviewAction>,
```

Comment:
i like this better than having loads of boolean columns.
the datamodel file is to be updated after this PR?